src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -1,7 +1,27 @@
 package lotto.view
 
+import lotto.Const
+import lotto.domain.LottoNumber
+
 class InputView {
```

Comment:
```suggestion
object InputView {
```
`object` 를 활용해 보면 어떨까요? 메시지와 관련된 상수들도 모아볼 수 있을것 같아요.